invenio_pidstore/models.py

```diff
@@ -352,60 +269,57 @@ def is_reserved(self):
         """Return true if the PID has not yet been reserved."""
         return self.status == PIDStatus.RESERVED
 
-    def log(self, action, message, id_pid_as_null=False):
-        """
-        Store action and message in log.
-
-        If 'id_pid_as_null' the foreign key to PID will not be set.
-        """
-        if self.pid_type and self.pid_value:
-            message = "[{0}:{1}] {2}".format(
-                self.pid_type, self.pid_value, message)
-        id_pid = None if id_pid_as_null else self.id
-        with db.session.begin_nested():
-            p = PidLog(id_pid=id_pid, action=action, message=message)
-            db.session.add(p)
-
+    def __repr__(self):
+        """Get representation of object."""
+        return "<PersistentIdentifier {0}:{1}{3} ({2})>".format(
+            self.pid_type, self.pid_value, self.status,
+            " / {0}:{1}".format(self.object_type, self.object_uuid) if
+            self.object_type else ""
+        )
 
-class PidLog(db.Model):
-    """Audit log of actions happening to persistent identifiers.
 
-    This model is primarily used through PersistentIdentifier.log and rarely
-    created manually.
-    """
+class Redirect(db.Model, Timestamp):
+    """Redirect for a persistent identifier."""
 
-    __tablename__ = 'pidLOG'
+    __tablename__ = 'pidstore_redirect'
     __table_args__ = (
-        db.Index('idx_action', 'action'),
+        db.ForeignKeyConstraint(
+            ['pid_type', 'pid_value'],
+            [PersistentIdentifier.pid_type, PersistentIdentifier.pid_value]),
     )
 
-    id = db.Column(
-        db.Integer,
-        primary_key=True
-    )
-    """Id of persistent identifier entry."""
+    id = db.Column(UUIDType, default=uuid.uuid4, primary_key=True)
+    """Id of redirect entry."""
 
-    id_pid = db.Column(
-        db.Integer,
-        db.ForeignKey(PersistentIdentifier.id),
-        nullable=True,
-    )
-    """PID."""
+    pid_type = db.Column(db.String(6), nullable=False)
+    """Persistent Identifier scheme."""
+
+    pid_value = db.Column(db.String(length=255), nullable=False)
```

Comment:
`db.String(255)` - or update https://github.com/inveniosoftware/invenio-pidstore/pull/31/files#diff-4a91ab23e2daf6cad1290964910b919dR294 for consistency too